### ISSUE TITLE
.im is not considered a TLD

### ISSUE THREAD

**Issue by metajack** [paulrouget/servoshell#59]
Typing "metajack.im" into the url bar takes me to duckduckgo instead of to the url. I'm guessing the TLD detection needs a bit of expansion.

**Comment by paulrouget** [comment]
relevant code: https://github.com/paulrouget/servoshell/blob/2235fb865cdecbefbb4ee108750411399b38ce8a/src/main.rs#L233

**Comment by paulrouget** [comment]
We should use `servo::net_traits::is_pub_domain`.

**Comment by tschneidereit** [comment]
> We should use `servo::net_traits::is_pub_domain`.

I agree - trying to properly handle TLDs in the embedding is a losing game - they're just too complex and require large-ish tables and such.

**Comment by paulrouget** [comment]
I'm confused, `is_pub_domain` seems to allow more that what `resources/pub_domain.txt` is listing.
